Gems/PhysX/Code/Source/Material/PhysXMaterial.cpp

```diff
@@ -153,6 +162,18 @@ namespace PhysX
         {
             return static_cast<AZ::u32>(GetFrictionCombineMode());
         }
+        else if (propertyName == MaterialConstants::CompliantContactModeEnabledName)
```

Comment:
>Would a switch work rather than all these else ifs?

It's a bit odd using switch with a string.

>And maybe AZ::Name instead of comparing string_views?

When creating the PhysX Material Asset I tried to use AZ::Name, they get serialized into the asset, but (at least at the time) the AZ:Name created at Asset Processor time didn't match the ones used at runtime, and I was having issues with AZ::Name created statically as well. So it was taken the decision to use strings at the time to move forward. But moving PhysX Materials Assets from using strings to AZ::Name is an optimization that we can try to do in the future.